src/builder.rs

```diff
@@ -0,0 +1,213 @@
+use crate::{
```

Comment:
The name builder for the module feels a bit too generic here, maybe we can call it something like "redaction_builder" instead?

Reply:
Decided to go for just `redactor` instead :)